lib/hexpm_web/templates/email/layout.text.eex

```diff
@@ -0,0 +1,4 @@
+<%= render @view_module, @view_template, assigns %>
+
+--
```

Comment:
```suggestion
-- 
```

https://tools.ietf.org/html/rfc3676#section-4.3

Reply:
Unfortunately we cannot make changes like this because most editors are configured to trim whitespace at the end of the line. Is this only a convention? What could happen if we do not add the space?

Reply:
If we don't add the space, e-mail readers won't format the signature as a signature (a different color, usually).

(I don't think editor configuration is preventing us from making that change. If a misconfigured editor messes this file up, just ignore that hunk during `git add -p`. Easy. :-))

Reply:
The issue isn't if it's possible to commit a change with trailing whitespace. But it is very easy to accidentally commit a change that reverts it without noticing.

Reply:
Let's do something like this instead: `<%= "-- " %>`

Reply:
That is indeed even better. Relying on humans not to make mistakes is indeed silly. :-)